### PR TITLE
Cetus & Earth Times, News & Daily Reward Reset Panels

### LINKED ISSUE #187: [Vue] News component
### Warframe Hub Issue
---

**Summary (short):**  News component in vue

- [x] Time posted
- [x] Title
- [x] Link
- [x] Image if possible

### LINKED ISSUE #191: [Vue] Fissures component
### Warframe Hub Issue
---

**Summary (short):**  Fissures component in vue

- [x] Fissure tier
- [x] Fissure time
- [x] Fissure node
- [x] Fissure mission type

### LINKED ISSUE #204: [Vue] Earth Times Component
### Warframe Hub Issue
---

**Summary (short):**

- [x] Time to next division
- [x] Time at next division
- [x] Badge for current division

### LINKED ISSUE #205: [Vue] Cetus cycle component
### Warframe Hub Issue
---

**Summary (short):**

- [x] Time to next division
- [x] Time at next division
- [x] Badge for current division

### PR DESCRIPTION
**Pull request by Bitti09**
### Warframe Hub Pull Request
---
**Summary (short):**
Add more components

---
**Description:**
Add:
- Cetus Time Component
- Earth Time Component
- News Component
- Daily Reset Panel

---
**Fixes issue (include link):**
closes #191
closes #205
closes #204
closes #187

---
**Mockups, screenshots, evidence:**

---

(Check one)
- [ ] Issue fix
- [x] Suggestion fulfillment

### CODE REVIEW

**Comment by TobiTenno** [comment]
Had an error on my end as part of the build, might need to rebase. sorry about that.

File: src/components/panels/ResetPanel.vue [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+<script>
+import TimeBadge from '@/components/TimeBadge.vue';
+import moment from 'moment';
+
+export default {
+  props: ['time', 'location'],
+  name: 'TimePanel',
+  computed: {
+    headertext() {
+      return 'Daily Reward Timer reset';
```
Comment (TobiTenno) [inline src/components/panels/ResetPanel.vue:24]: Title case? `Daily Reward Timer Reset` or even simply `Daily Reset`

File: src/components/panels/ResetPanel.vue [hunk 2]
```diff
@@ -0,0 +1,55 @@
+<template>
+<b-col md="6">
+  <h3 class="text-center">{{headertext}}</h3>
+  <b-list-group>
+    <b-list-group-item :style="styleObject" class="list-group-item-borderless">
```
Comment (TobiTenno) [inline src/components/panels/ResetPanel.vue:5]: could you use `list-group-item-borderbottom`?

File: src/components/panels/NewsPanel.vue [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  color: white;
+  width: 100% !important;
+  left: 0;
+  bottom: 0;
+  font-weight: bolder;
+  padding-bottom: 0;
+}
+.carousel-caption a {
+  color: white;
+}
```
Comment (TobiTenno) [inline src/components/panels/NewsPanel.vue:29]: might want to toy with the img styling so it the image inside stays it's original aspect ratio?

**Comment by TobiTenno** [comment]
this is more for me for future reference after it's merged:
```css
.timers {
    margin-top: 70px;
    margin-bottom: 45px;
}

/* General class to add to the panels */
.hub-panel {
    margin-bottom: 30px;
}
```

**Comment by TobiTenno** [comment]
Looks like the night badge is missing styling

**Comment by EricSihaoLin** [comment]
Decided to check out the build, it seems like the news component isn't working too well on Safari.
screen shot 2018-11-02 at 1 28 55 am
That or Safari is just not good. (I apologize as I'm currently in the Mac ecosystem.)
The spacing also seems inconsistent, but that could just be because it is early in development.

**Comment by TobiTenno** [comment]
It's also not going multi-column on chrome or edge

chrome:
image

edge:
image

**Comment by TobiTenno** [comment]
@EricSihaoLin the spacing is because we don't have packery or something to replace it as in the previous site

**Pull request closed without merging**